.gitignore

```diff
@@ -0,0 +1,12 @@
+/**/__pycache__/
+/**/output.sas
+/**/sas_plan*
```

Comment:
I generally prefer keeping ignores as local as possible to minimize false positives. In this case, I would limit these to the root directory (which contains fast-downward.py) and to ignore sas_plan and sas_plan.[1-9]* rather than all files everywhere whose base name begins with sas_plan.

Reply:
I agree.